pkg/apis/app/v1alpha1/workspace_types.go

```diff
@@ -17,6 +17,20 @@ type Module struct {
 	Version string `json:"version"`
 }
 
+// VCS holds all the information needed to connect the workspace to a VCS repository
+type VCS struct {
+	// Token ID of the VCS Connection (OAuth Connection Token) to use
+	TokenID string `json:"token_id"`
+	// A reference to your VCS repository in the format org/repo
+	RepoIdentifier string `json:"repo_identifier"`
+	// The repository branch to use
+	// +optional
+	Branch string `json:"branch"`
+	// Whether submodules should be fetched when cloning the VCS repository (Defaults to false)
+	// +kubebuilder:default=false
+	IngressSubmodules bool `json:"ingress_submodules"`
```

Comment:
This might need an `omit:empty`, since the CR required this field during testing.